--- ./xwobf.c	2019-01-03 01:43:55.531413182 -0500
+++ ./xwobf.c	2019-01-03 01:51:36.251726593 -0500
@@ -26,7 +26,7 @@
 #include <getopt.h>
 #include <err.h>
 #include <xcb/xcb.h>
-#include <MagickWand/MagickWand.h>
+#include <wand/MagickWand.h>
 
 #include "xwobf.h"
 
@@ -191,14 +191,14 @@
 
         // This is where the magick happens
         (void)MagickResizeImage(obs_wand, (rec->w)/pixel_size, (rec->h)/pixel_size,
-                PointFilter);
+                PointFilter, 0);
         if (fuzzy) {
                 (void)MagickBlurImage(obs_wand, 0, 1);
         }
         (void)MagickResizeImage(obs_wand, rec->w, rec->h,
-                PointFilter);
+                PointFilter, 0);
 
-        (void)MagickCompositeImage(wand, obs_wand, OverCompositeOp, MagickTrue, rec->x, rec->y);
+        (void)MagickCompositeImage(wand, obs_wand, OverCompositeOp, rec->x, rec->y);
 
         obs_wand = DestroyMagickWand(obs_wand);
     }
